apps/tsilhqotin-language-hub/src/Components/Widgets/Footer/Footer.tsx

```diff
@@ -102,9 +102,16 @@ export function Footer() {
             <Divider className="footerDivider" />
             <div className="copyrightContainer">
                 <div className="copyright">
-                    © 2022 Tŝilhqot’in National Government.{' '}
-                    <span id="rights">All Rights Reserved.</span>
+                    {' '}
+                    <img
+                        src="https://www.tsilhqotin.ca/wp-content/uploads/2022/11/tl_logo_blue_transparent.png"
```

Comment:
I suppose we can go ahead and hold off on pulling data from configs in the stand-alone projects for now.